apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -472,37 +472,111 @@ int ble_rmc_main(int argc, char *argv[])
 		printf("\n");
 	}
 
+	if (strncmp(argv[1], "whitelist", 10) == 0) {
+		if (argc == 4 && strncmp(argv[2], "add", 4) == 0) {
+			ble_addr addr[1] = { 0, };
+			int count = 0;
+
+			count = sscanf(argv[3], "%02x:%02x:%02x:%02x:%02x:%02x", &addr->mac[0], &addr->mac[1], &addr->mac[2],
+				&addr->mac[3], &addr->mac[4], &addr->mac[5]);
+			if (count != BLE_BD_ADDR_MAX_LEN) {
+				RMC_LOG(RMC_CLIENT_TAG, "Fail to read MAC[%d]\n", count);
+				goto ble_rmc_done;
+			}
+
+			RMC_LOG(RMC_CLIENT_TAG, "Input Mac[%d] : %02x:%02x:%02x:%02x:%02x:%02x\n", addr->type, addr->mac[0], 
+				addr->mac[1], addr->mac[2], addr->mac[3], addr->mac[4], addr->mac[5]);
+
+			ret = ble_scan_whitelist_add(addr);
+			if (ret != BLE_MANAGER_SUCCESS) {
+				RMC_LOG(RMC_CLIENT_TAG, "Add whitelist fail[%d]\n", ret);
+				goto ble_rmc_done;
+			}
+			RMC_LOG(RMC_CLIENT_TAG, "Add whitelist Success\n");
+		} else if (argc == 4 && strncmp(argv[2], "del", 4) == 0) {
+			ble_addr addr[1] = { 0, };
+			int count = 0;
+
+			count = sscanf(argv[3], "%02x:%02x:%02x:%02x:%02x:%02x", &addr->mac[0], &addr->mac[1], &addr->mac[2],
+				&addr->mac[3], &addr->mac[4], &addr->mac[5]);
+			if (count != BLE_BD_ADDR_MAX_LEN) {
+				RMC_LOG(RMC_CLIENT_TAG, "Fail to read MAC[%d]\n", count);
+				goto ble_rmc_done;
+			}
+
+			RMC_LOG(RMC_CLIENT_TAG, "Input Mac[%d] : %02x:%02x:%02x:%02x:%02x:%02x\n", addr->type, addr->mac[0], 
+				addr->mac[1], addr->mac[2], addr->mac[3], addr->mac[4], addr->mac[5]);
+
+			ret = ble_scan_whitelist_delete(addr);
+			if (ret != BLE_MANAGER_SUCCESS) {
+				RMC_LOG(RMC_CLIENT_TAG, "Del whitelist fail[%d]\n", ret);
+				goto ble_rmc_done;
+			}
+			RMC_LOG(RMC_CLIENT_TAG, "Del whitelist Success\n");
+		} else if (argc == 3 && strncmp(argv[2], "clear", 6) == 0) {
+			ret = ble_scan_whitelist_clear_all();
+			if (ret != BLE_MANAGER_SUCCESS) {
+				RMC_LOG(RMC_CLIENT_TAG, "Clear whitelist fail[%d]\n", ret);
+				goto ble_rmc_done;
+			}
+			RMC_LOG(RMC_CLIENT_TAG, "Clear whitelist Success\n");
+		} else if (argc == 3 && strncmp(argv[2], "list", 5) == 0) {
+			ble_addr addr_list[10] = { 0, };
+			ble_addr *addr;
+			uint16_t count = 0;
+			int i;
+			count = ble_scan_whitelist_list(addr_list, 10);
+
+			RMC_LOG(RMC_CLIENT_TAG, "Total List : %d\n", count);
```

Comment:
`ble_scan_whitelist_list` returns the unsigned value, so I think it is better to print the `count` with `%u`.

Reply:
ok. I will fix it